.gitmodules

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: 2021 Settlers Freaks <sf-team at siedler25.org>
```

Comment:
same here `2005 - 2021`

Reply:
That is "no license" anyway, so doesn't really matter, and similar issue as above: Would need to find when this was added and not fully sure if you can add a copyright notice for something published earlier and date that back

Reply:
of course we can date it back, since its not about the file, but about the package, so everything under our copyright in combination with s25client is starting from 2005

Reply:
Done